sanic/pages/styles/ErrorPage.css

```diff
@@ -1,2 +1,52 @@
 /** ErrorPage **/
-summary { color: #888; }
+summary {
+    color: #888;
+}
+
+.tracerite {
+    /* --color-var: var(--accent);
+    --color-val: #ff0; */
+    --color-type: var(--accent);
+}
```

Comment:
Nice touch, I assume this is the Sanic logo red color?

Note: needs a change in tracerite as well (move from :root to .tracerite and rename from --tracerite-type back to --color-type).

Reply:
Yes, accent being the base Sanic color. We really only have the one, but if ever there was a time to create a bit of a palette, now is the chance.

Reply:
@Tronic 

1. No need to move the variables in tracerite from `:root`. We should still be able to override them okay in `.tracerite` in this project. I suggest they remain there.
2. I also think tracerite should continue to  namespace its variables as `--tracerite-*`. No need to convert back to `--color-*`.